include/exiv2/types.hpp

```diff
@@ -102,12 +102,12 @@ namespace Exiv2 {
         asciiString        = 2, //!< Exif ASCII type, 8-bit byte.
         unsignedShort      = 3, //!< Exif SHORT type, 16-bit (2-byte) unsigned integer.
         unsignedLong       = 4, //!< Exif LONG type, 32-bit (4-byte) unsigned integer.
-        unsignedRational   = 5, //!< Exif RATIONAL type, two LONGs: numerator and denumerator of a fraction.
+        unsignedRational   = 5, //!< Exif RATIONAL type, two LONGs: numerator and denominator of a fraction.
```

Comment:
Sometimes I run into projects that are trying to do something special here. But I think this is right.